gemini/use-cases/retrieval-augmented-generation/rag_vector_embedding_in_bigquery.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        CONN_SERVICE_ACCOUNT = f"serviceAccount:{request.cloud_resource.service_account_id}"
</pre><p>This probably shouldn't be in all caps (const) since it's being reassigned later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>